spec/brainstem/api_docs/formatters/open_api_specification/version_2/endpoint/param_definitions_formatter_spec.rb

```diff
@@ -34,38 +34,86 @@ module Endpoint
               end
 
               describe '#call' do
-                context 'when request type is get' do
-                  let(:http_methods)  { %w(GET) }
+                context 'when an endpoint presents a brainstem object' do
+                  context 'when request type is get' do
+                    let(:http_methods)  { %w(GET) }
+
+                    context 'when action is index' do
+                      let(:action) { 'index' }
+
+                      it 'formats path, shared, query and body params for the endpoint' do
+                        any_instance_of(described_class) do |instance|
+                          mock(instance).format_path_params!
+                          mock(instance).format_optional_params!
+                          mock(instance).format_include_params!
+                          mock(instance).format_query_params!
+                          mock(instance).format_body_params!
+                          mock(instance).format_pagination_params!
+                          mock(instance).format_search_param!
+                          mock(instance).format_only_param!
+                          mock(instance).format_sort_order_params!
+                          mock(instance).format_filter_params!
+                        end
+
+                        subject.call
+                      end
+                    end
 
-                  context 'when action is index' do
-                    let(:action) { 'index' }
+                    context 'when action is show' do
+                      let(:action) { 'show' }
+
+                      it 'formats path, optional, query and body params for the endpoint' do
```

Comment:
Commit Message: `Only add query and body params for endpoints that do not use a presenter to return data`

The commit message is correct. The intention was to say only query & body params are added to endpoints that do not use a brainstem presenter. It would have been better to also call out that `pagination`, `includes` & `only` (shared params) are not included on endpoints that do not use a brainstem presenter.